lib/nexmo_developer/app/services/translator/translator_coordinator.rb

```diff
@@ -0,0 +1,23 @@
+module Translator
+  class TranslatorCoordinator
+    attr_accessor :paths
+
+    def initialize(params = {})
+      @paths = params.fetch(:paths)
```

Comment:
set a default value here so that it doesn't raise,
```
params.fetch(:paths, [])
```

Reply:
What if I do want it to raise, though? The benefit of `#fetch` is it won't let it be empty. Do we want to allow a new instance with no paths?

Reply:
there's no need to raise here a bit of defensive programming doesn't hurt, if `paths` is  an empty array the rest of the methods will still work